src/camera.cpp

```diff
@@ -498,7 +498,8 @@ void Camera::setDigging(s32 button)
 
 void Camera::wield(const ItemStack &item)
 {
-	if (item.name != m_wield_item_next.name) {
+	if (item.name != m_wield_item_next.name
+		|| item.metadata != m_wield_item_next.metadata) {
```

Comment:
Should be indented one more time, and the line break should be after the binary op (`||`)